src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -454,6 +455,11 @@ private void TraceFailingBranch(in ProcessingBranch processingBranch, Processing
             catch (InvalidBlockException ex)
             {
                 BlockTraceDumper.LogDiagnosticTrace(blockTracer, ex.InvalidBlock.Hash!, _logger);
+                Metrics.BadBlocks++;
+                if (Encoding.ASCII.GetString(ex.InvalidBlock.ExtraData) == "Nethermind")
```

Comment:
Instead of hardcoding `Nethermind` we  might want to get the default value from `Nethermind.Config.BlocksConfig`

Reply:
I would go with both. Also would use Contains and InvariantCultureIgnoreCase comparision.

Reply:
@LukaszRozmej I changed to contains and ignoring case. Are you saying I should hardcode `Nethermind` as well as getting it from the config - why would that be better than just using the config?